src/index.ts

```diff
@@ -96,7 +96,7 @@ export default async function (manager: Manager, settings: ComponentSettings) {
   })
 
   manager.addEventListener('pageview', event => {
-    const eventType = event.payload.ev
+    const eventType = event.payload.ev || 'PAGE_VIEW'
     handler(eventType, event, settings)
```

Comment:
since this is the `pageview` handler - shouldn't we just do something like this?

```suggestion
    handler("PAGE_VIEW", event, settings)
```

I mean, it cannot really be anything other than a pageview.